src/org/opensolaris/opengrok/configuration/Configuration.java

```diff
@@ -143,6 +143,10 @@
     private Set<Group> groups;
     private String sourceRoot;
     private String dataRoot;
+    /**
+     * directory with include files for web application (header, footer, etc.)
+     */
+    private String includeRoot;
```

Comment:
Wouldn't be better to use `Path`? However, I'm not sure if the serialization will work...

Reply:
I will rather keep it the same as sourceRoot/dataRoot.